### DISCUSSION TITLE
User permission

### DISCUSSION THREAD

**Discussion by stealink** [mozilla/hubs#4956] [Hubs Cloud Questions] [answered]
Hello, 
I want to allow access to a room to certain users but not to others . 
EX : 
I have room A with user 1 and I don't want user 1 have access to room B but the user need to login before .
thx for your reply

**Comment by pattersonbl2** [comment, marked as answer]
We currently don't have this feature. I will open an issue on your behalf with the team to have it added to the board.

**Comment by matthewbcool** [comment]
@stealink If you wish to make a feature request on hubs-cloud, please provide more details and create a new issue here: https://github.com/mozilla/hubs-cloud/issues

using the following template: 
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.

**Additional context**
Add any other context or screenshots about the feature request here.
